### ISSUE TITLE
Render only Header

### ISSUE THREAD

**Issue by ashrithks** [react-native-community/react-native-tab-view#264]
Hi all,
Just a question,
How to render only header part

**Comment by bsonntag** [comment]
You mean the `TabBar` component? You can probably just import it and use it:

```js
import { TabBar } from 'react-native-tab-view';

function SomeComponent() {
  return <TabBar />;
}
```

You'll have to take a look at the props it receives from `TabViewAnimated` if it starts complaining about missing props. You can see those props [here](https://github.com/react-native-community/react-native-tab-view/blob/master/src/TabViewAnimated.js#L202).

**Comment by satya164** [comment]
Hey, I just released a new alpha `2.0.0-alpha.0` of the library. It's rewritten using [react-native-gesture-handler](https://github.com/kmagiera/react-native-gesture-handler) and [react-native-reanimated](https://github.com/kmagiera/react-native-reanimated) addresses a many platform specific bugs and performance problems. The documentation is updated as well.

Please try the new version and see if it addresses your issue. If not, please open a new issue following the issue template.
